src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.prejap.ess.helpers.arrays;
+
+import java.util.Arrays;
+
+/**
+ * Examples below shown differences between fill and setAll methods.
```

Comment:
```suggestion
 * Examples below shown differences between {@link Arrays#fill(Object[], Object)} and {@link Arrays#setAll(Object[], IntFunction)} methods.
```
In cases like that use `{@link ...}` to provide a direct link to a given javadoc. Someone may use it to just jump there and read what is needed.

Reply:
To be more precise a direct link to the members of a class.